projects/adrastea/index.js

```diff
@@ -0,0 +1,52 @@
+const { Connection, PublicKey } = require('@solana/web3.js');
```

Comment:
you can instead do this like:

    const { sumTokensExport } = require('../helper/solana')
    module.exports = {
      solana: {
        tvl: sumTokensExport({ owners: ['Ec5tJ1H24iVSM2L8Yd7SHf7bjtD7FUWDiYSeESpFYynM'] })
      }
    }